source/crud/tests/unified/updateMany-sort.yml

```diff
@@ -0,0 +1,117 @@
+description: updateMany-sort
+schemaVersion: "1.0"
+createEntities:
+  - client:
+      id: client0
+      observeEvents:
+        - commandStartedEvent
+  - database:
+      id: database0
+      client: client0
+      databaseName: crud-tests
+  - collection:
+      id: collection0
+      database: database0
+      collectionName: coll0
+initialData:
+  - collectionName: coll0
+    databaseName: crud-tests
+    documents:
+      - _id: 1
+        x: 11
+      - _id: 2
+        x: 22
+      - _id: 3
+        x: 33
+tests:
+  - description: UpdateMany with sort option unsupported
+    runOnRequirements:
+      - minServerVersion: "8.0"
+    operations:
+      - name: updateMany
+        object: collection0
+        arguments:
+          filter:
+            _id:
+              $gt: 1
+          sort:
+            _id: -1
+          update:
+            $inc:
+              x: 1
+        expectError:
+          isError: true
+    expectEvents:
+      - client: client0
+        events:
+          - commandStartedEvent:
```

Comment:
Wouldn't a client-side error preclude sending a command to the server? Consider asserting that `events` is empty here. That may be done in other spec tests for client-side errors.

Likewise, I don't think there's any need for an `outcome` assertion if we know we're not sending anything to the server.